perun-core/src/test/java/cz/metacentrum/perun/core/entry/AttributesManagerEntryIntegrationTest.java

```diff
@@ -792,7 +792,7 @@ public void getAllResourceAttributesStartWithNameWithoutNullValue() throws Excep
 			attributesManager.setAttribute(sess, resource, a);
 		}
 
-		List<Attribute> retAttr = attributesManager.getAllAttributesStartWithNameWithoutNullValue(sess, resource, AttributesManager.NS_RESOURCE_ATTR_OPT + ":resource_test_uniqueattribute:");
+		List<Attribute> retAttr = attributesManager.getAllAttributesStartWithNameWithoutNullValue(sess, resource, AttributesManager.NS_RESOURCE_ATTR_OPT + ":specialNamespace");
```

Comment:
The name is ok, just change character "_" to "-".
Method getAllAttributesStartWithNameWithoutNullValue looking for attributes starting with full name
like "urn:perun:resource:attribute-def:opt:resource_test_uniqueattribute:". Friendly name is there just "resource_test_uniqueattribute:", not the delimiter ":" before that.